TileStache/Providers.py

```diff
@@ -77,14 +77,14 @@
     from io import BytesIO
 except ImportError:
     # Python 2
-    from StringIO import StringIO as BytesIO
+    from io import StringIO as BytesIO
 from string import Template
 try:
     import urllib.request as urllib2
 except ImportError:
     # Python 2
-    import urllib2
-import urllib
+    import urllib.request, urllib.error, urllib.parse
+import urllib.request, urllib.parse, urllib.error
```

Comment:
try/except here... but, do we really need both urllib and urllib2? My code inspector says we are not actually using urllib